collectors/gsuite/collector.js

```diff
@@ -124,6 +124,10 @@ class GsuiteCollector extends PawsCollector {
                         });
                     }
                     else {
+                        // set errorCode if not available in error object to showcase client error on DDMetrics
+                        if (error.errors && error.errors.length > 0 && error.errors[0].reason) {
+                            error.errorCode = error.errors[0].reason;
```

Comment:
Is that going to be short enough to work as an error code?

Reply:
yes, It is short and also explain error.  The reason we are getting is like 'insufficientPermissions' . We are also getting code (ex.403). Shall I use code ?

Reply:
As the reason field is  short and more granular than the http code and we had already used reason field  for checking throttling error as well. To keep code consistent, using reason field as error code.